### PR TITLE
refactor: remove deprecated methods and adjust code

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Description and Example

remove deprecated methods and adjust code to use correct methods.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.
- [x] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle-android/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/NavigationBarViewScreenBuilder.kt [hunk 1]
```diff
@@ -18,8 +18,7 @@ package br.com.zup.beagle.sample.builder
 
 import br.com.zup.beagle.core.Accessibility
 import br.com.zup.beagle.core.Style
```
Comment (botbeaglezup) [inline sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/NavigationBarViewScreenBuilder.kt]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/6a9db095ea62c806b2f4e00e3f31ce02b3210ca8/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/NavigationBarViewScreenBuilder.kt#L20"></span>The import 'br.com.zup.beagle.core.Style' is unused.</td>
    </tr>
  </tbody>
</table>

File: sample/spring/src/main/kotlin/br/com/zup/beagle/sample/spring/service/SampleNavigationTypeService.kt [hunk 2]
```diff
@@ -18,8 +18,7 @@ package br.com.zup.beagle.sample.spring.service
 
 import br.com.zup.beagle.core.CornerRadius
 import br.com.zup.beagle.core.Style
```
Comment (botbeaglezup) [inline sample/spring/src/main/kotlin/br/com/zup/beagle/sample/spring/service/SampleNavigationTypeService.kt]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/6a9db095ea62c806b2f4e00e3f31ce02b3210ca8/sample/spring/src/main/kotlin/br/com/zup/beagle/sample/spring/service/SampleNavigationTypeService.kt#L20"></span>The import 'br.com.zup.beagle.core.Style' is unused.</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-backend-kotlin/blob/b2ede46dae081f3fd943db0aa8e6d6633f8e3d3f/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt#L42">sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt#L42</a> - The import 'br.com.zup.beagle.sample.constants.SCREEN_FORM_ENDPOINT' is unused.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-backend-kotlin/blob/b2ede46dae081f3fd943db0aa8e6d6633f8e3d3f/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt#L57">sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt#L57</a> - The import 'br.com.zup.beagle.sample.constants.SCREEN_TAB_VIEW_ENDPOINT' is unused.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-backend-kotlin/blob/b2ede46dae081f3fd943db0aa8e6d6633f8e3d3f/sample/spring/src/main/kotlin/br/com/zup/beagle/sample/spring/controller/ScreenController.kt#L56">sample/spring/src/main/kotlin/br/com/zup/beagle/sample/spring/controller/ScreenController.kt#L56</a> - The import 'br.com.zup.beagle.sample.constants.SCREEN_TAB_VIEW_ENDPOINT' is unused.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_backend">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by matheusribeirozup — APPROVED** [review]

**Pull request merged**